### PR TITLE
Update carousel.md

### PR DESCRIPTION
**Pull request by manonthemat**
Clarify that carousel's interval is set in milliseconds.

### CODE REVIEW

File: site/content/docs/5.0/components/carousel.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td><code>interval</code></td>
       <td>number</td>
       <td><code>5000</code></td>
-      <td>The amount of time to delay between automatically cycling an item. If false, carousel will not automatically cycle.</td>
+      <td>The amount of time in ms to delay between automatically cycling an item. If false, carousel will not automatically cycle.</td>
```
Comment (XhmikosR) [inline site/content/docs/5.0/components/carousel.md:289]: @mdo or `in milliseconds`, your call. We should also backport this.

Comment (mdo) [inline site/content/docs/5.0/components/carousel.md:289]:
```suggestion
      <td>The amount of time in milliseconds to delay between automatically cycling an item. If false, carousel will not automatically cycle.</td>
```

**Comment by XhmikosR** [comment]
@mdo I think we need to change all instances like in popovers and toasts too, so that we are consistent.

@manonthemat can you take care of it please?

File: site/content/docs/5.0/components/carousel.md (hunk 1 above)
Comment (patrickhlauke) [reply site/content/docs/5.0/components/carousel.md:289]: counter-counter proposal: "the amount of delay (in ms) before cycling an item"

**Comment by manonthemat** [comment]
> @manonthemat can you take care of it please?

Will do.

**Comment by XhmikosR** [comment]
@manonthemat can you also change the other instances in docs?

**Comment by XhmikosR** [comment]
Closing due to lack of response. You can comment here and we can re-open the PR, if you decide to resume working on the requested changes.

*This is an automated reply*

**Pull request closed without merging**